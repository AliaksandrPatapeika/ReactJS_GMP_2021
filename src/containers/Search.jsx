import '../less/containers/Search.less';

import React from 'react';

import SearchButton from '../components/SearchButton';
import SearchInput from '../components/SearchInput';

const Search = () => (
  <div className="searchContainer">
    <SearchInput />
    <SearchButton />
  </div>
);

export default Search;
