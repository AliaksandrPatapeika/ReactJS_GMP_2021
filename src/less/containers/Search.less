.searchContainer {
    display: flex;
    margin-top: 33px;
}
