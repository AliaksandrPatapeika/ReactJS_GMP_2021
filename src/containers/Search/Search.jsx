import './Search.less';

import React from 'react';

import Button from '../../components/Button';
import SearchInput from '../../components/SearchInput';
import {MockOnClick} from '../../mocks/mockData';

const Search = () => (
  <div className="searchContainer">
    <SearchInput />
    <Button className="searchButton" onClick={() => MockOnClick('searchButton')} title="SEARCH" />
  </div>
);

export default Search;
