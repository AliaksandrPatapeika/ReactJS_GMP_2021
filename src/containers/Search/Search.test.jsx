import React from 'react';
import renderer from 'react-test-renderer';

import Search from './Search';

describe('Search', () => {
  test('renders correctly', () => {
    const search = renderer
      .create(<Search />)
      .toJSON();

    expect(search).toMatchSnapshot();
  });
});
