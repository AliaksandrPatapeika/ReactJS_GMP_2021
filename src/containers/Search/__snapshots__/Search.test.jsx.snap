// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<div
  className="searchContainer"
>
  <input
    className="searchInput"
    placeholder="What do you want to watch?"
  />
  <button
    className="searchButton"
    onClick={[Function]}
    type="button"
  >
    SEARCH
    
  </button>
</div>
`;
